PERF_GUIDE.md

```diff
@@ -0,0 +1,53 @@
+# Build performance
+
+When we talk about build performance, it is important to understand that there
+are several build types:
+
++ cold build (booting your app up for the first time)
```

Comment:
elaborate on differences between build types; set expectations on build times; if app doesn't meet expectations, there might be a problem; how do we identify the problem and what can tools help us with it;